lib/source-destination/multi-destination.ts

```diff
@@ -108,9 +114,21 @@ export class MultiDestination extends SourceDestination {
 		if (destinations.length === 0) {
 			throw new Error('At least one destination is required');
 		}
-		destinations.map((destination: SourceDestination) => {
+		for (const destination of destinations) {
 			this.destinations.add(destination);
-		});
+		}
+	}
+
+	public getAlignment(): number | undefined {
+		const alignments: number[] = [];
+		for (const destination of this.destinations.values()) {
+			if (destination instanceof BlockDevice) {
+				alignments.push(destination.alignment);
+			}
+		}
+		if (alignments.length) {
+			return Math.max(...alignments);
+		}
```

Comment:
This can be
```suggestion
		return _.maxBy(this.destinations.values(), () => {
			if (destination instanceof BlockDevice) {
				return destination.alignment;
			}
		});
```

Reply:
I didn't change this one as:
 * this would require adding an `Array.from` as lodash does not work with iterators;
 * `maxBy` would return the BlockDevice itself, not the alignment;

Reply:
replaced with 
```
	public getAlignment(): number | undefined {
		let max = undefined;
		for (const destination of this.destinations.values()) {
			if ((destination instanceof BlockDevice) && (max === undefined || destination.alignment > max)) {
				max = destination.alignment;
			}
		}
		return max;
	}
```

Reply:
Actually to be 100% correct, this should return the least common multiple of all alignments.
But as the only alignment I've seen are 512 and 4096, max also works.

Reply:
You could possibly start with `let max = 0;` and avoid the `undefined` check

Reply:
A 0 `alignment` would make no sense.

Reply:
I prefer keeping it as `undefined` => don't align.

Reply:
You could do it as `if (alignment === 0) return undefined` but I'm fine with the version you posted in that case as well